components/pages/people/case-status/view/case-status.spec.tsx

```diff
@@ -0,0 +1,76 @@
+import { render } from '@testing-library/react';
+import * as caseStatusApi from 'utils/api/caseStatus';
+import { mockedAPIservererror } from 'factories/APIerrors';
+import CaseStatusView from './case-status';
+
+import {
+  mockedCaseStatusFactory,
+  mockedPersonCaseStatusFactory,
+} from 'factories/caseStatus';
+import { mockedResident } from 'factories/residents';
+
+jest.mock('next/router', () => ({
+  useRouter: () => ({
+    asPath: 'path',
+    push: jest.fn(),
+  }),
+  reload: jest.fn(),
+}));
+
+jest.mock('components/Spinner/Spinner', () => () => 'MockedSpinner');
+
+const person = mockedResident;
+
+describe('CaseStatusView component', () => {
+  it('displays the casestatus of a person', async () => {
+    jest.spyOn(caseStatusApi, 'GetCaseStatus').mockImplementation(() => ({
+      data: mockedPersonCaseStatusFactory.build({
+        personId: person.id,
+        caseStatuses: [
+          mockedCaseStatusFactory.build({
+            type: 'CIN',
+          }),
+        ],
+      }),
+      isValidating: false,
+      mutate: jest.fn(),
+      revalidate: jest.fn(),
+    }));
+
+    const { getByText } = render(<CaseStatusView person={person} />);
+
+    expect(getByText('CIN')).toBeInTheDocument();
+  });
+
+  it('displays "No case status found for this person" if no case status', async () => {
+    jest.spyOn(caseStatusApi, 'GetCaseStatus').mockImplementation(() => ({
+      data: mockedPersonCaseStatusFactory.build({
+        personId: person.id,
+        caseStatuses: [],
+      }),
+      isValidating: false,
+      mutate: jest.fn(),
+      revalidate: jest.fn(),
+    }));
+
+    const { queryByText } = render(<CaseStatusView person={person} />);
+
+    expect(queryByText(/CIN/)).not.toBeInTheDocument();
+  });
```

Comment:
The test description is different to the expectation we have. Are we expecting to just not see any tags i.e. no "CIN" or a "No case status found for this person"?